projects/observability/src/shared/dashboard/dashboard-wrapper/navigable-dashboard.component.ts

```diff
@@ -74,7 +77,9 @@ export class NavigableDashboardComponent implements OnChanges {
     if (changeObject.navLocation) {
       const persistedDashboard$ =
         this.navLocation !== undefined && this.navLocation !== null
-          ? this.dashboardPersistenceService.getForLocation(this.navLocation)
+          ? this.customJson
+            ? this.dashboardPersistenceService.getForLocation(this.navLocation)
+            : this.dashboardPersistenceService.getDefaultForLocation(this.navLocation)
```

Comment:
Nested ternaries are often confusing to read. 
https://eslint.org/docs/rules/no-nested-ternary

Can we extract this bit to a separate variable and use that here for better readability?

Reply:
Noted